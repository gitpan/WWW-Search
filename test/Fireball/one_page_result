http://www.fireball.de/query-fireball.fcg?what=german_web&action=query&fmt=d&pg=express&q=%2BAnna+%2BKournikova+%2BWimbledon+%2BWTA+%2Btennis
