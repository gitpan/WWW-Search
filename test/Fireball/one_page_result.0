<html>
<head>
<title>FIREBALL</title>
</head>
<body link="#CC0000" vlink="#600000" bgcolor="#FFFFFF">
<a NAME="top"></a>
<center>
<!-- obere Formatierungstabelle -->
<MAP NAME="obenmap">
<AREA SHAPE="rect" ALT="Home" COORDS="0,0,55,20" HREF="express.html">
<AREA SHAPE="rect" ALT="Detail Suche" COORDS="57,0,154,20" HREF="detail.html">
<AREA SHAPE="rect" ALT="Live Suche" COORDS="155,0,237,20" HREF="voyeur-fireball.fcg?action=voyeur-queries" target="_top">
<AREA SHAPE="rect" ALT="Info und Tools" COORDS="238,1,331,20" HREF="intern.html">
<AREA SHAPE="rect" ALT="URL melden" COORDS="332,0,418,20" HREF="url_melden.html">
<AREA SHAPE="rect" ALT="Hilfe" COORDS="419,0,471,20" HREF="express_hilfe.html">
</MAP>
<table cellspacing="0" cellpadding="0" bgcolor="#CC0000" border="0">
<tr>
<td width="25"><img src="bilder/verschiedenes/1.gif" alt="" width="25" height="0"></td>
<td bgcolor="#FFFFCC">
<table border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="100%"><img src="bilder/leisten/menubar.gif" width="471" height="21" usemap="#obenmap" alt="FIREBALL Express Suche" border="0"></td>
</tr>
<tr>
<td width="100%" height="2">
<!-- gelber Index-->
<table border="0" cellpadding="0" cellspacing="0" width="100%" bgcolor="#FFFFCC">
<tr>
<td width="12%" align="center" bgcolor="#FFCC00">&nbsp; </td>
<td width="21%" align="center">&nbsp; </td>
<td width="18%" align="center">&nbsp; </td>
<td width="19%" align="center">&nbsp; </td>
<td width="19%" align="center">&nbsp; </td>
<td width="11%" align="center">&nbsp; </td>
</tr>
</table>
</td>
</tr>
</table>
<!--- Eingabefelder --->
<table border="0" cellpadding="3" cellspacing="0" width="471" bgcolor="#FFFFCC">
<tr>
<td>
<FORM METHOD="GET" ACTION="/query-fireball.fcg">
<INPUT TYPE="HIDDEN" NAME="action" VALUE="query">
<INPUT NAME="pg" TYPE="HIDDEN" VALUE="express">
	<table border="0" cellpadding="3" cellspacing="0" width="454">
<tr>
<td align="right"><b><font face="arial,helvetica" size="1">Nach: </font></b></td>
<td align="left" colspan="3" width="387">
<INPUT TYPE="text" name="q" VALUE="+Anna +Kournikova +Wimbledon +WTA +tennis" SIZE=30>&nbsp;
<input TYPE="submit" VALUE="suchen"></td>
</tr>
<tr>
<td align="right" width="54"><b><font face="arial,helvetica" size="1">Sprache: </font></b></td>
<td align="left" colspan="2" width="212"><SELECT name="what">
			<OPTION VALUE="german_web" SELECTED>deutsch
			<OPTION VALUE="web" >allen Sprachen (AltaVista)
			<OPTION VALUE="da" >d&auml;nisch (AltaVista)
			<OPTION VALUE="en" >englisch (AltaVista)
			<OPTION VALUE="et" >estnisch (AltaVista)
			<OPTION VALUE="fi" >finnisch (AltaVista)
			<OPTION VALUE="fr" >franz&ouml;sisch (AltaVista)
			<OPTION VALUE="nl" >holl&auml;ndisch (AltaVista)
			<OPTION VALUE="is" >isl&auml;ndisch (AltaVista)
			<OPTION VALUE="it" >italienisch (AltaVista)
			<OPTION VALUE="lv" >lettisch (AltaVista)
			<OPTION VALUE="lt" >lithauisch (AltaVista)
			<OPTION VALUE="no" >norwegisch (AltaVista)
			<OPTION VALUE="pl" >polnisch (AltaVista)
			<OPTION VALUE="pt" >portugiesisch (AltaVista)
			<OPTION VALUE="ro" >rum&auml;nisch (AltaVista)
			<OPTION VALUE="sv" >schwedisch (AltaVista)
			<OPTION VALUE="es" >spanisch (AltaVista)
			<OPTION VALUE="cz" >tschechisch (AltaVista)
			<OPTION VALUE="hu" >ungarisch (AltaVista)
		    </SELECT></td>
<td rowspan="2" bgcolor="#FFFFCC" width="170"><img src="bilder/leisten/logo02.gif" width=179 height=59 border="0" alt="Fireball Logo" align="right"></td>
</tr>
<tr>
<td align="right" width="54"><b><font face="arial,helvetica" size="1">Anzeige: </font></b></td>
<td align="left" width="158"><SELECT name="fmt">
			<OPTION VALUE="d" SELECTED>detail
			<OPTION VALUE="." >standard
			<OPTION VALUE="c" >kompakt
		    </SELECT></td>
<td align="center" width="48"><b><font face="arial,helvetica" size="1"><a href="express_hilfe_kurz.html">Tips:</a> </font></b></td>
</tr>
</table>
</form>
</td>
</tr>
</table>
</td>
</tr>
</table>
<!-- tips -->
<br>
<table BORDER="0" CELLPADDING="2" CELLSPACING="0" WIDTH="496" align="center">
  <tr>
    <td valign="top" WIDTH="25" bgcolor="#CC0000"><img SRC="bilder/verschiedenes/1.gif" alt="" BORDER="0" width="1"
    height="1"></td>
    <td bgcolor="#FFFFCC">&nbsp;<A HREF="/forward-fireball.fcg?action=sforward&furl=http%3A%2F%2Fwww.boulevard.de%2Findex.phtml%3Fsc%3D10015001%26aktion%3Dprofisuche%26schlagwort%3DAnna%2BKournikova%2BWimbledon%2BWTA%2Btennis&rurl=%2Fquery%2Dfireball%2Dplus%2Dstart%2D1%2Efcg%3Fwhat%3Dgerman%5Fweb%26action%3Dquery%26fmt%3Dd%26pg%3Dexpress%26q%3D%252BAnna%2B%252BKournikova%2B%252BWimbledon%2B%252BWTA%2B%252Btennis" target="_blank"><font face="arial,helvetica">Hier finden Sie Bücher zu Ihrer Suchanfrage.</font></a></td>
	<td bgcolor="#FFFFCC" align="right"><A HREF="/forward-fireball.fcg?action=sforward&furl=http%3A%2F%2Fwww.boulevard.de%2Findex.phtml%3Fsc%3D10015001%26aktion%3Dprofisuche%26schlagwort%3DAnna%2BKournikova%2BWimbledon%2BWTA%2Btennis&rurl=%2Fquery%2Dfireball%2Dplus%2Dstart%2D1%2Efcg%3Fwhat%3Dgerman%5Fweb%26action%3Dquery%26fmt%3Dd%26pg%3Dexpress%26q%3D%252BAnna%2B%252BKournikova%2B%252BWimbledon%2B%252BWTA%2B%252Btennis" target="_blank"><img src="bilder/werbung/bol_68_29.gif" border="0" alt="Boulevard Online" width="68" height="29"></a></td>
  </tr>
  
  <tr>
    <td valign="top" WIDTH="25"><img src="bilder/verschiedenes/1.gif" alt="" width="21" height="1"></td>
    <td></td>
	<td></td>
  </tr>
<tr>
<td valign="top" WIDTH="25" bgcolor="#CC0000"><img SRC="bilder/verschiedenes/1.gif" alt="" BORDER="0" width="1" height="1"></td>
<td bgcolor="#FFFFCC">&nbsp;<A HREF="/forward-fireball.fcg?action=sforward&furl=http%3A%2F%2Fwww.cd4you.de&rurl=%2Fquery%2Dfireball%2Dplus%2Dstart%2D1%2Efcg%3Fwhat%3Dgerman%5Fweb%26action%3Dquery%26fmt%3Dd%26pg%3Dexpress%26q%3D%252BAnna%2B%252BKournikova%2B%252BWimbledon%2B%252BWTA%2B%252Btennis" target="_blank"><font face="arial,helvetica">CDs zu Ihrer Musikanfrage gibt es hier:</font></a></td>
<td bgcolor="#FFFFCC" align="right"><A HREF="/forward-fireball.fcg?action=sforward&furl=http%3A%2F%2Fwww.cd4you.de&rurl=%2Fquery%2Dfireball%2Dplus%2Dstart%2D1%2Efcg%3Fwhat%3Dgerman%5Fweb%26action%3Dquery%26fmt%3Dd%26pg%3Dexpress%26q%3D%252BAnna%2B%252BKournikova%2B%252BWimbledon%2B%252BWTA%2B%252Btennis" target="_blank"><img src="bilder/werbung/Primus-Promo-cd-68x29-261198.gif" border="0" alt="Boulevard Online" width="68" height="29"></a></td>
</tr>

  <tr>
    <td valign="top" WIDTH="25"><img src="bilder/verschiedenes/1.gif" alt="" width="21" height="1"></td>
    <td></td>
	<td></td>
  </tr>
  <tr>
    <td valign="top" WIDTH="25" bgcolor="#CC0000">&nbsp; </td>
    <td bgcolor="#FFFFCC" colspan="2">&nbsp;<font face="arial,helvetica">Weitere Infos in <a href="http://www.fireball.de/surftips/art/spo.class.html">surftips/sport/classic sports</a></font></td>
  </tr>
</table>
<!--- ende tips--->
<!--- werbung --->
<br>
<table border="0" cellpadding="0" cellspacing="0">
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;<font face="arial,helvetica" size="1">Anzeige&nbsp;&nbsp;</font></td>
<td>

<!--- advertisement stuff (forward-daemon, banner, zmod-daemon, zmod-module) - context-sensitive ad-engine developed by FLP/KIT --->
<A HREF="/forward-fireball.fcg?action=forward&furl=http%3A%2F%2Fwww%2Eiqanzeigen%2Ede%2Fems%2Ehbs&rurl=CP%2Fquery%2Dfireball%2Dplus%2Dstart%2D1%2Efcg&adID=2263&imageName=%2Fwerbung%2FTVT%2Diq%2D468%2D071298%2Egif&matchReason=express&customerName=538"><IMG SRC="/werbung/TVT-iq-468-071298.gif" ALT="Anzeige" BORDER=0></A>
<IMG SRC="/zmod-fireball.fcg?page=CP%2Fquery%2Dfireball%2Dplus%2Dstart%2D1%2Efcg&ad=2263&imageName=%2Fwerbung%2FTVT%2Diq%2D468%2D071298%2Egif&matchReason=express&customerName=538&info=localQuery#" WIDTH=1 HEIGHT=1>
<IMG SRC="/cgi-bin/vdz/CP/query-fireball-plus-start-1.fcg/2263%3A/werbung/TVT-iq-468-071298.gif%3Aexpress%3A538" WIDTH=1 HEIGHT=1>
<!--- end of advertisement stuff --->

</td>
</tr>
</table>
<!--- werbung ende beginn tip und libri --->

<br>
<!--- tabelle head suchergebnis --->
<table border="0" cellpadding="2" cellspacing="0" width="496">
<tr>
	<td bgcolor="#CC0000" width="25">&nbsp; </td>
  <td bgcolor="#CC0000"><font face="arial,helvetica" size="2" color="#FFFFFF"><b>Suchergebnis:</b></font></td>
	<td bgcolor="#CC0000">&nbsp; </td>
  <td bgcolor="#CC0000">&nbsp; </td>
  <td width="100">&nbsp; </td>
</tr>

<tr>
   <td bgcolor="#CC0000" width="25"><img src="bilder/verschiedenes/1.gif" alt="" width="21" height="1"></td>
   <td bgcolor="#FFCC00"><font FACE="Arial,Helvetica" SIZE="1">&nbsp;Zu viele Treffer? Suche <a href="detail.html">verfeinern?</a></font></td>
   <td bgcolor="#FFFFCC">&nbsp; </td>
   <td bgcolor="#FFFFCC">&nbsp; </td>
   <td bgcolor="#FFFFCC" width="100">&nbsp; </td>
 </tr>

  <tr>
        <td bgcolor="#CC0000" width="25">&nbsp; </td>
        <td bgcolor="#FFFFCC" colspan="4" rowspan="2"><font FACE="Arial,Helvetica" SIZE="2">
	<!--- term statistics --->
        &nbsp;
<!--- term statistics --->
<FONT FACE="Arial,Helvetica" SIZE=-1>Worth&auml;ufigkeiten: Kournikova: 597; WTA: 4123; Wimbledon: 5363; Anna: 87659; tennis: 95324</FONT>
<!--- end term statistics --->


	<!--- end term statistics ---> <br>
        <b><!--- hit count information --->&nbsp;
<!--- hit count information --->
Dokument 1-5 von 5 Treffern, beste Treffer zuerst.
<!--- end hit count information --->


	<!--- end hit count information ---> </b></font></td>
      </tr>

      <tr>
        <td width="25">&nbsp; </td>
      </tr>
 </table>
<br>
<!--- Ende Teil 1 und tabelle head suchergebnis--->
<!--- Teil 2 -- Ergebnisausgabe --->
<table BORDER="0" CELLPADDING="2" CELLSPACING="0" WIDTH="496">
  <tr>
    <td WIDTH="25"></td>
    <td></td>
  </tr>
<!--- hits --->
  <!--- separator --->

<!--- separator --->


<!--- hits --->
<TR><td valign="top" WIDTH="25" bgcolor="#CC0000"><img SRC="bilder/verschiedenes/1.gif" BORDER="0" width="25" height="1">&nbsp;</td><td bgcolor="#FFFFCC"><b><a HREF="http://www.dsf.de/a5/headlines/5334/bginfo.html"><font FACE="Arial,Helvetica" SIZE="2">Eastbourne: Schimpfende Graf verliert gegen Kournikova</font></a></b></td></tr><tr><td valign="top" WIDTH="25"><img src="bilder/verschiedenes/1.gif" width="21" height="1"></td><td><font FACE="Arial,Helvetica" SIZE="2">E-MAIL IMPRESSUM   Eastbourne: Schimpfende Graf verliert gegen Kournikova   Eastbourne, 18.06.98. Eine lauthals schimpfende Steffi Graf (Bild) ist beim  WTA-Turnier in Eastbourne im Viertelfinale ausgeschieden. Doch trotz der  7:6 (7:4), 3:6, 4:6-Ni<br></font><font FACE="Arial,Helvetica" SIZE="1"><a HREF="http://www.dsf.de/a5/headlines/5334/bginfo.html">http://www.dsf.de/a5/headlines/5334/bginfo.html</a><BR><i>Gr&ouml;&szlig;e 10 K - 18.6.1998  </i><br><br></font></td></tr>

<TR><td valign="top" WIDTH="25" bgcolor="#CC0000"><img SRC="bilder/verschiedenes/1.gif" BORDER="0" width="25" height="1">&nbsp;</td><td bgcolor="#FFFFCC"><b><a HREF="http://www.oon.at/generali-ladies/kournikova.html"><font FACE="Arial,Helvetica" SIZE="2">Generali Ladies Linz 1998 - Anna Kournikova</font></a></b></td></tr><tr><td valign="top" WIDTH="25"><img src="bilder/verschiedenes/1.gif" width="21" height="1"></td><td><font FACE="Arial,Helvetica" SIZE="2">präsentiert: Generali  Ladies  Das ist der Tennis-Hammer! Kournikova kommt nach Linz   Das ist ein echter "Hammer" für alle Tennisfans: Die Russin Anna Kournikova  kommt zum "Generali-Ladies presented by Zipfer und Raiffeisen" vom 21.  Februar bis 1<br></font><font FACE="Arial,Helvetica" SIZE="1"><a HREF="http://www.oon.at/generali-ladies/kournikova.html">http://www.oon.at/generali-ladies/kournikova.html</a><BR><i>Gr&ouml;&szlig;e 4 K - 8.4.1998  </i><br><br></font></td></tr>

<TR><td valign="top" WIDTH="25" bgcolor="#CC0000"><img SRC="bilder/verschiedenes/1.gif" BORDER="0" width="25" height="1">&nbsp;</td><td bgcolor="#FFFFCC"><b><a HREF="http://www.mopo.de/dcweb/client/mopo/11.5.98/ressort/sport/sportlinks.html"><font FACE="Arial,Helvetica" SIZE="2">MOPO-Sportlinks</font></a></b></td></tr><tr><td valign="top" WIDTH="25"><img src="bilder/verschiedenes/1.gif" width="21" height="1"></td><td><font FACE="Arial,Helvetica" SIZE="2">Sport im World Wide Web<br></font><font FACE="Arial,Helvetica" SIZE="1"><a HREF="http://www.mopo.de/dcweb/client/mopo/11.5.98/ressort/sport/sportlinks.html">http://www.mopo.de/dcweb/client/mopo/11.5.98/ressort/sport/sportlinks.html</a><BR><i>Gr&ouml;&szlig;e 8 K - 10.5.1998  </i><br><br></font></td></tr>

<TR><td valign="top" WIDTH="25" bgcolor="#CC0000"><img SRC="bilder/verschiedenes/1.gif" BORDER="0" width="25" height="1">&nbsp;</td><td bgcolor="#FFFFCC"><b><a HREF="http://www.mopo.de/dcweb/client/mopo/30.5.98/ressort/sport/sportlinks.html"><font FACE="Arial,Helvetica" SIZE="2">MOPO-Sportlinks</font></a></b></td></tr><tr><td valign="top" WIDTH="25"><img src="bilder/verschiedenes/1.gif" width="21" height="1"></td><td><font FACE="Arial,Helvetica" SIZE="2">Sport im World Wide Web<br></font><font FACE="Arial,Helvetica" SIZE="1"><a HREF="http://www.mopo.de/dcweb/client/mopo/30.5.98/ressort/sport/sportlinks.html">http://www.mopo.de/dcweb/client/mopo/30.5.98/ressort/sport/sportlinks.html</a><BR><i>Gr&ouml;&szlig;e 9 K - 29.5.1998  </i><br><br></font></td></tr>

<TR><td valign="top" WIDTH="25" bgcolor="#CC0000"><img SRC="bilder/verschiedenes/1.gif" BORDER="0" width="25" height="1">&nbsp;</td><td bgcolor="#FFFFCC"><b><a HREF="http://ourworld.compuserve.com/Homepages/d_munkel/gesagt.htm"><font FACE="Arial,Helvetica" SIZE="2">Tennis Compress Gesagt ist gesagt</font></a></b></td></tr><tr><td valign="top" WIDTH="25"><img src="bilder/verschiedenes/1.gif" width="21" height="1"></td><td><font FACE="Arial,Helvetica" SIZE="2">TENNIS Compress  Gesagt ist gesagt  zur Homepage  Thomas Muster nach seiner Niederlage gegen Filip Dewulf in Kitzbühel über  die Pfiffe der Zuschauer:  "Wennst so schlecht spielst, ist das wohl ganz normal."  Nicola Pilic in Anspielung auf Boris Bec<br></font><font FACE="Arial,Helvetica" SIZE="1"><a HREF="http://ourworld.compuserve.com/Homepages/d_munkel/gesagt.htm">http://ourworld.compuserve.com/Homepages/d_munkel/gesagt.htm</a><BR><i>Gr&ouml;&szlig;e 16 K - 12.9.1998  </i><br><br></font></td></tr>


<!--- end hits --->


<!--- end hits --->
</table>
<table border="0" cellpadding="0" cellspacing="0" width="496">
  <tr>
    <td width="25"></td>
    <td colspan="4">&nbsp; </td>
    <td>&nbsp; </td>
  </tr>
  <tr>
    <td width="25">&nbsp; </td>
    <td bgcolor="#FFFFCC" colspan="5" rowspan="2"><table border="0" width="471">
<!--- result page references --->


<!--- end result page references --->

</table></td>
  </tr>
  <tr>
    <td width="25" bgcolor="#CC0000"><img src="bilder/verschiedenes/1.gif" alt="" width="21" height="1">&nbsp; </td>
  </tr>
  <tr>
    <td width="25" bgcolor="#CC0000">&nbsp; </td>
    <td bgcolor="#FFCC00"><font face="arial,helvetica" size="2"><b>&nbsp;&nbsp;<a href="#top">Seitenanfang</a></b></font></td>
    <td bgcolor="#FFCC00"><font face="arial,helvetica" size="2"><b></b></font></td>
    <td bgcolor="#FFCC00"><font face="arial,helvetica" size="2"><b> </b></font></td>
    <td bgcolor="#FFFFCC">&nbsp; </td>
    <td bgcolor="#FFFFCC">&nbsp; </td>
  </tr>
  <tr>
    <td width="25" bgcolor="#CC0000"><img src="bilder/verschiedenes/1.gif" alt="" width="25" height="1"></td>
    <td bgcolor="#CC0000" colspan="3">&nbsp; </td>
    <td bgcolor="#CC0000">&nbsp; </td>
    <td>&nbsp; </td>
  </tr>
</table>
<br>
<!--- Ende Teil 2 --->

<!--- footer container --->
<!--- werbung footer --->
<table BORDER="0" CELLPADDING="0" CELLSPACING="0">
  <tr>
    <td><font face="arial,helvetica" size="1">Anzeige&nbsp; </font></td>
    <td WIDTH="518">

<!--- (last) advertisement stuff (forward-daemon, banner) --->
<A HREF="/forward-fireball.fcg?action=forward&furl=http%3A%2F%2Fwww%2Eiqanzeigen%2Ede%2Fems%2Ehbs&rurl=CP%2Fquery%2Dfireball%2Dplus%2Dstart%2D1%2Efcg&adID=2263&imageName=%2Fwerbung%2FTVT%2Diq%2D468%2D071298%2Egif&matchReason=express&customerName=538"><IMG SRC="/werbung/TVT-iq-468-071298.gif" BORDER=0></A>
<!--- end of (last) advertisement stuff --->

</td>
  </tr>
</table>
<!--- werbung footer ende --->
<!--- footer container --->

<table border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td valign="top">
<!--- linke tabelle --->
	<table border="0" cellpadding="0"
    cellspacing="0" width="85%">
      <tr>
        <td width="25"></td>
        <td colspan="2">&nbsp; </td>
        <td>&nbsp; </td>
        <td>&nbsp; </td>
      </tr>
      <tr>
        <td width="25">&nbsp; </td>
        <td bgcolor="#FFFFCC" colspan="4" rowspan="2">
		<table border="0" cellpadding="3" cellspacing="0">
          <tr>
            <td width="50%" nowrap><font face="arial,helvetica" size="2"><a
            href="http://www.fireball.de/surftips/art/neu.html">Neu</a> <br>
            <a href="http://www.fireball.de/surftips/art/au.idx.html">Auskunft</a>&nbsp; <br>
            <a href="http://www.fireball.de/surftips/art/co.idx.html">Computer</a>&nbsp; <br>
            <a href="http://www.fireball.de/surftips/art/ge.idx.html">Geld &amp; Karriere</a>&nbsp; <br>
            <a href="http://www.fireball.de/surftips/art/in.idx.html">Internet</a>&nbsp; <br>
            <a href="http://www.fireball.de/surftips/art/ki.idx.html">Kino</a>&nbsp; <br>
            <a href="http://www.fireball.de/surftips/art/wi.idx.html">Kultur &amp;&nbsp; <br>
            &nbsp; Wissenschaft</a>&nbsp; <br>
            <a href="http://www.fireball.de/surftips/art/le.idx.html">Leib &amp; Seele</a></font></td>
            <td width="50%" nowrap><font face="arial,helvetica" size="2"><a
            href="http://www.fireball.de/surftips/art/li.idx.html">Liebe &amp; Erotik</a><br>
            <a href="http://www.fireball.de/surftips/art/me.idx.html">Medien</a><br>
            <a href="http://www.fireball.de/surftips/art/mo.idx.html">Mode &amp;</a><br>
            <a href="http://www.fireball.de/surftips/art/mo.idx.html">Shopping</a><br>
            <a href="http://www.fireball.de/surftips/art/mu.idx.html">Musik</a><br>
            <a href="http://www.fireball.de/surftips/art/re.idx.html">Reisen</a><br>
            <a href="http://www.fireball.de/surftips/art/spi.idx.html">Spiel &amp; Spaß</a><br>
            <a href="http://www.fireball.de/surftips/art/spo.idx.html">Sport</a><br>
            <a href="http://www.fireball.de/surftips/art/wr.idx.html">Wirtschaft</a></font></td>
          </tr>
        </table>
        </td>
      </tr>
      <tr>
        <td width="25" bgcolor="#CC0000"><img src="bilder/verschiedenes/1.gif" alt="" width="21" height="1">&nbsp; </td>
      </tr>
      <tr>
        <td width="25" bgcolor="#CC0000">&nbsp; </td>
        <td bgcolor="#FFCC00">&nbsp; <font face="arial, helvetica" size="1">Best of Internet.</font></td>
        <td bgcolor="#FFFFCC">&nbsp; </td>
        <td bgcolor="#FFFFCC">&nbsp; </td>
        <td bgcolor="#FFFFCC">&nbsp; </td>
      </tr>
      <tr>
        <td width="25" bgcolor="#CC0000">&nbsp; </td>
        <td bgcolor="#CC0000"><font face="arial,helvetica" size="2" color="#FFFFFF"><b>&nbsp;
        Surftips</b></font></td>
        <td bgcolor="#CC0000">&nbsp; </td>
        <td>&nbsp; </td>
        <td>&nbsp; </td>
      </tr>
    </table>
</td>
    <td>
<!-- disclaimer rahmen -->
	<table border="0" cellpadding="2" cellspacing="0">
      <tr>
        <td width="25">&nbsp; </td>
        <td colspan="3">&nbsp; <img src="bilder/verschiedenes/1.gif" alt="" width="1" height="25"></td>
        <td>&nbsp; </td>
      </tr>
      <tr>
        <td width="25">&nbsp; </td>
        <td rowspan="2" bgcolor="#FFFFCC" colspan="3" align="right">
<!-- disclaimer innen -->
		<table border="0" cellpadding="3" cellspacing="0">
          <tr>
            <td nowrap align="right"><font face="arial,helvetica" size="2">Powered by<br>
            <a HREF="http://altavista.software.digital.com/altv/index.htm" target="_blank"><img
            SRC="bilder/werbung/av.gif" BORDER="0" ALIGN="bottom" alt="AltaVista" width="59" height="22"><br>
            Search Software</a><br>
            und<br>
            <a HREF="http://www.digital.de/products/Alphaserver/Alphaserver.html" target="_blank">Digital
            AlphaServer</a><br><br>
			Offizieller Suchpartner<br>
            von </font><a href="http://www.t-online.de/"><img src="bilder/werbung/t_online.gif" width="77"
            height="13" alt="t-online" border="0"></a></td>
            <td nowrap align="right"><font face="arial, helvetica" size="2"><a HREF="detail.html"
            target="_top">Detail Suche</a><br>
            <a HREF="voyeur-fireball.fcg?action=voyeur-queries" target="_top">Live Suche</a><br>
			<a HREF="express_t.html" target="_top">Textversion</a><br>
            <a HREF="intern.html" target="_top">Info &amp; Tools</a><br>
            <a HREF="url_melden.html" target="_top">URL melden</a><br>
            <a HREF="guestbook-fireball.fcg" target="_top">Gästebuch</a><br>
            <a HREF="express_hilfe.html" target="_top"><b>Hilfe</b></a><br>
            <a HREF="impressum.html">Impressum</a><br>
            <a HREF="kontakt.html">Kontakt</a><br>
            <a HREF="nutzungsordnung.html">Nutzungs-<br>
            ordnung</a></font></td>
          </tr>
        </table>
		</td>
        <td>&nbsp; </td>
      </tr>
      <tr>
        <td width="25"><img src="bilder/verschiedenes/1.gif" alt="" width="21" height="1">&nbsp; </td>
        <td bgcolor="#CC0000">&nbsp; </td>
      </tr>
      <tr>
        <td width="25">&nbsp; </td>
        <td bgcolor="#FFFFCC">&nbsp; </td>
        <td bgcolor="#FFFFCC">&nbsp; </td>
        <td bgcolor="#FFCC00" align="right"><font face="arial,helvetica" size="1">© <a
        href="http://www.ems.guj.de">G+J Electronic Media Service</a>, <br>
        Hamburg 1997/98.</font></td>
        <td bgcolor="#CC0000">&nbsp; </td>
      </tr>
      <tr>
        <td width="25">&nbsp; </td>
        <td align="right">&nbsp; </td>
        <td bgcolor="#CC0000">&nbsp; </td>
        <td bgcolor="#CC0000" align="right"><font face="arial,helvetica" size="2" color="#FFFFFF"><b>Fireball</b></font></td>
        <td bgcolor="#CC0000"><img src="bilder/verschiedenes/1.gif" alt="" width="21" height="1"></td>
      </tr>
    </table>
	 </td>
  </tr>
</table>
</center>
</body>
</html>

