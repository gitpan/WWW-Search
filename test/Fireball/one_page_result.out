http://www.dsf.de/a5/headlines/5334/bginfo.html
http://www.oon.at/generali-ladies/kournikova.html
http://www.mopo.de/dcweb/client/mopo/11.5.98/ressort/sport/sportlinks.html
http://www.mopo.de/dcweb/client/mopo/30.5.98/ressort/sport/sportlinks.html
http://ourworld.compuserve.com/Homepages/d_munkel/gesagt.htm
